src/foam/core/reflow/Console.js

```diff
@@ -32,7 +32,7 @@ foam.CLASS({
     { name: 'value' },
     {
       name: 'treeRowRenderer',
-      value: function(e) { e.add(this.flowName); }
+      value: function(e) { this.flowName == 'flow' ? e.add('Flow') : e.add(this.flowName); }
```

Comment:
I disagree because when users are trying to access properties from the flow then they'll think that the variable name is 'Flow', but it isn't.